static/src/javascripts/projects/commercial/modules/prebid/bid-config.spec.js

```diff
@@ -730,6 +563,7 @@ describe('bids', () => {
 
     test('should include AppNexus directly if in target geolocation', () => {
         shouldIncludeAppNexus.mockReturnValue(true);
+        isInAuRegion.mockReturnValue(true);
```

Comment:
Adding a mockReturnValue here since appNexusBidder now needs to check the region:

```js
bidParams: (slotId: string, sizes: PrebidSize[]): PrebidAppNexusParams =>
        getAppNexusDirectBidParams(sizes, isInAuRegion()),
```